python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -1827,6 +1888,16 @@ def run(self, ccdExposure, *, dnlLUT=None, bias=None, deferredChargeCalib=None,
         keywords = self.config.cameraKeywordsToCompare
         if not self.config.doBootstrap:
             compareCameraKeywords(doRaise, keywords, exposureMetadata, ptc, "PTC", log=self.log)
+            # Note that doCorrectGains can be True without a gainCorrection.
```

Comment:
Something should warn in this case.  If we have the config on, but can't do the correction, that shouldn't succeed without a log message.  Either here, or below at L2073 where it does this check again.

Reply:
I'd appreciate an info log (I know I like logs more than most).

Reply:
After thinking about this I realized that (a) the code was missing setting `LSST ISR GAINCORRECTION APPLIED`, and (b) below at the application time an info log would be useful as you say.